ide/coqide/coqide.ml

```diff
@@ -1132,6 +1394,12 @@ let build_ui () =
       ];
     item "Show Proof Diffs" ~label:"_Show Proof (with diffs, if set)" ~accel:(modifier_for_display#get ^ "S")
       ~callback:MiscMenu.show_proof_diffs;
+    (* todo: move the following somewhere better *)
+    item "Toggle breakpoint" ~label:"_Toggle breakpoint" ~accel:"F8"
+      ~callback:MiscMenu.toggle_breakpoint;
+    item "Continue" ~label:"_Continue" ~accel:"F9" ~callback:Nav.continue;
+    item "Step in" ~label:"Step in" ~accel:"F10" ~callback:Nav.step_in;
+    item "Step out" ~label:"Step out" ~accel:"<Shift>F10" ~callback:Nav.step_out;
```

Comment:
This is not visible, and there is no scroll bar
![image](https://user-images.githubusercontent.com/396076/118325964-404f4200-b4d2-11eb-854e-7d24f991ee4b.png)